backend/inspirehep/records/marshmallow/authors/ui.py

```diff
@@ -58,6 +59,32 @@ def get_facet_author_name(self, data):
     def get_twitter(data):
         return get_first_value_for_schema(data.get("ids", []), "TWITTER")
 
+    @staticmethod
+    def get_all_urls_field(data):
+
+        urls = data.get("urls", [])
+        author_profiles = [
```

Comment:
Can we change naming there to `external_author_profiles`? It is a bit confusing with author profile on INSPIRE imho